test/sql/test_utils.py

```diff
@@ -174,3 +175,15 @@ def test_unwrap_order_by(self, expr, expected):
 
         for a, b in zip_longest(unwrapped, expected):
             assert a is not None and a.compare(b)
+
+    def test_column_collection_get(self):
```

Comment:
there didn't appear to be any tests for this method (I looked by all references), and this misc location seemed as appropriate as any. Happy to relocate somewhere else if more appropriate.